addons/account_analytic_default/models/account_analytic_default.py

```diff
@@ -83,5 +83,10 @@ def _compute_analytic_account(self):
                 date=record.date_maturity,
                 company_id=record.move_id.company_id.id
             )
-            record.analytic_account_id = (record._origin or record).analytic_account_id or rec.analytic_id
-            record.analytic_tag_ids = (record._origin or record).analytic_tag_ids or rec.analytic_tag_ids
+            partner = record.partner_id.commercial_partner_id or record.move_id.partner_id.commercial_partner_id
+            if not partner or record.account_id.user_type_id.type not in ('receivable', 'payable'):
```

Comment:
What's the purpose of the check on partner?

Besides, I realized that if you switch partner to the one linked to the analytic default setting, tags are applied also on tax lines, we should fix also that
```python
if (not partner or record.account_id.user_type_id.type not in ('receivable', 'payable')) and not record.tax_line_id:
```